Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -4127,6 +4130,35 @@ def gcb_list_user_aliases(client_obj: Client, start_time: str, end_time: str, pa
     return ec, json_data
 
 
+def gcb_verify_rule(client_obj: Client, rule_text: str):
+    """
+    Return context data and raw response for gcb_verify_rule command.
+
+    :type client_obj: Client
+    :param client_obj: client object which is used to get response from api.
+
+    :type rule_text: str
+    :param rule_text: items to validate.
+
+    :rtype: Tuple[Dict[str, Any], Dict[str, Any]]
+    :return: ec, json_data: Context data and raw response of the request.
+    """
+    req_json_data = {
+        'ruleText': rule_text
+    }
+    request_url = f"{BACKSTORY_API_V2_URL}/detect/rules:verifyRule"
+    json_data = validate_response(client_obj, request_url, method='POST', body=json.dumps(req_json_data))
+    context_data = deepcopy(json_data)
+    context_data['success'] = json_data.get('success', False)
+    context_data['command_name'] = 'gcb-verify-rule'
+
+    ec = {
+        CHRONICLE_OUTPUT_PATHS['VerifyRule']: context_data
+    }
```

Comment:
```suggestion
    context_data = {
        **json_data,
        'success': json_data.get('success', False),
        'command_name': 'gcb-verify-rule',
    }

    ec = {CHRONICLE_OUTPUT_PATHS['VerifyRule']: context_data}
```

You can also achieve it this way